frontend/src/pages/ActivityReport/Pages/activitySummary.js

```diff
@@ -602,13 +604,27 @@ export const isPageComplete = (formData, formState) => {
 
   const numbersToValidate = [
     duration,
-    numberOfParticipants,
   ];
 
   if (!numbersToValidate.every((num) => num && Number.isNaN(num) === false)) {
     return false;
   }
 
+  // Handle custom validation for number of participants.
+  let participantsToValidate = [];
+  if (deliveryMethod === 'hybrid') {
+    participantsToValidate = [
+      numberOfParticipantsInPerson,
+      numberOfParticipantsVirtually,
+    ];
+  } else {
+    participantsToValidate = [numberOfParticipants];
+  }
+
+  if (!participantsToValidate.every((num) => num && Number.isNaN(num) === false)) {
```

Comment:
since 0 is falsy this check will reject 0 as valid - is that okay?

Reply:
Your saying 0 would fail validation, correct? If yes, I think that's correct (it was like that before) also why would they select hybrid if they only had people attending one way and not hybrid?